Sources/Build/BuildOperation.swift

```diff
@@ -862,6 +797,48 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
         return (buildDescription, buildManifest)
     }
 
+    private func detectUnhandledFiles(
```

Comment:
Missing a doc comment, I would like to see it elaborate on what "handling" means in this context.

```suggestion
    /// 
    private func detectUnhandledFiles(
```

Reply:
Renamed to `diagnoseUnhandledFiles` and added a comment.